src/main/java/pl/ug/virtualofficebackend/domain/user/entity/User.java

```diff
@@ -0,0 +1,14 @@
+package pl.ug.virtualofficebackend.domain.user.entity;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
```

Comment:
Nie lepiej docelowo trzymac Id userow jako UUID?

Reply:
Możemy tez jako UUID, ale wtedy nie wiem, jak będzie działać GeneratedValue, czy w ogóle będzie.
I czemu własnie nie long?

Reply:
UUID.random() moze generowac nam nowe uuid i bedzie to tez jakis rodzaj security ktory mozemy pokazac przy prezentacji

Reply:
Jako long mozemy spokojnie trzymac jakies przedmioty/pokoje zeby ulatwic dostep do tego ale imo dane usera traktowac jako wrażliwe dane

Reply:
Szczerze nie widzę większych korzyści zabezpieczających z skorzystania z UUID zamiast z id, ale mogę się na to zgodzić

Reply:
uzywajac long mozesz sobie przechodzic po userach wpisujac po kolei 1,2,3,4,5 etc. Przy uzywaniu UUID nie bedzie to juz takie latwe